GitHubReactions.ps1

```diff
@@ -432,12 +587,29 @@ filter Add-GitHubReactionAdditionalProperties
 <#
     .SYNOPSIS
         Adds type name and additional properties to ease pipelining to GitHub Reaction objects.
+
     .PARAMETER InputObject
         The GitHub object to add additional properties to.
+
     .PARAMETER TypeName
         The type that should be assigned to the object.
+
+    .PARAMETER OwnerName
+        Owner of the repository.
+        If not supplied here, the DefaultOwnerName configuration property value will be used.
+
+    .PARAMETER RepositoryName
+        Name of the repository.
+        If not supplied here, the DefaultRepositoryName configuration property value will be used.
+
+    .PARAMETER Issue
```

Comment:
This description is no longer accurate, and you need to add a `.PARAMETER PullRequest` as well.